torch/fx/experimental/symbolic_shapes.py

```diff
@@ -274,7 +274,7 @@ def has_free_symbols(val: Union[SymInt, torch.Tensor]) -> bool:
 # Like free_symbols, but filtered to only report unbacked symbols
 def free_unbacked_symbols(x):
     # NB: keep synced with is_unbacked_symint
-    return {s for s in free_symbols(x) if s.name.startswith(("i", "f"))}
+    return {s for s in free_symbols(x) if s.name.startswith(("u", "f"))}
```

Comment:
Should there be / is there some documentation in this file on what the various possible starting chars in a symbol name mean? (I didn't know there were so many.)

Reply:
I would write it but even I don't know what they all are 😭